sdk/nodejs/output.ts

```diff
@@ -216,7 +216,7 @@ This function may throw in a future version of @pulumi/pulumi.`;
         };
 
         return new Proxy(this, {
-            get: (obj, prop: keyof T) => {
+            get: (obj, prop: string) => {
```

Comment:
This was failing with:
```
Type '(obj: this, prop: keyof T) => any' is not assignable to type '(target: this, p: string | symbol, receiver: any) => any'.
  Types of parameters 'prop' and 'p' are incompatible.
    Type 'string | symbol' is not assignable to type 'keyof T'.
      Type 'string' is not assignable to type 'keyof T'.

```

Not sure this is the right fix though, any ideas?

Reply:
Or just leave off the type altogether so that it's inferred as `string | symbol` from https://github.com/microsoft/TypeScript/blob/0996bff509d08c48280b87622937db722f25c827/lib/lib.es2015.proxy.d.ts#L26 (we do check `typeof prop === "string"` and `typeof prop === "symbol"` below)